spec/spec_helper.rb

```diff
@@ -3,6 +3,10 @@
 require 'database_cleaner'
 require 'factory_girl'
 require 'dotenv'
+require 'capybara/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.